.travis.yml

```diff
@@ -68,13 +69,11 @@ install:
   - "travis_retry pip install -e .[${EXTRAS}]"
 
 before_script:
-  - "export DISPLAY=:99.0" #set up xvfb, req'd for non-phantomJS webdrivers
-  - "sh -e /etc/init.d/xvfb start"
-  - "export E2E_WEBDRIVER_BROWSERS='Firefox'"
-  - "export E2E_WEBDRIVER_TIMEOUT='300'" # Max allowed time for E2E tests
+  - "if [ ${E2E} == 'yes' ]; then export DISPLAY=:99.0; sh -e /etc/init.d/xvfb start; export E2E_WEBDRIVER_BROWSERS='Firefox'; export E2E_WEBDRIVER_TIMEOUT='300'; fi"
```

Comment:
That was the original plan, but for some reason it didn't work on Travis.
I wanted to check if it'd work like this (which it did), then forgot to remove the `e2e_setup.sh` file, apparently.